CHANGELOG.unreleased.md

```diff
@@ -28,3 +28,4 @@ Section headings should be at level 3 (e.g. `### Added`).
 - Correct the artifact url for organization registry artifacts to be independent of the artifact type (@ibindlish in https://github.com/wandb/wandb/pull/10049)
 - Suffixes on sanitized `InternalArtifact` names have been shortened to 6 alphanumeric characters (@tonyyli-wandb in https://github.com/wandb/wandb/pull/10102)
 - `wandb.Video` will not print a progress spinner while encoding video when `WANDB_SILENT`/`WANDB_QUIET` environment variables are set (@jacobromero in https://github.com/wandb/wandb/pull/10064)
+- Registries fetched using `api.registries()` had an extra `wandb-registry-` prefix (@estellazx in https://github.com/wandb/wandb/pull/10187)
```

Comment:
The sentence should describe the new state, like "Good thing now happens" or "Bad thing no longer happens", see the other ones for examples. Could you update this?